src/catalogue/items/catalogueItemsTable.component.test.tsx

```diff
@@ -904,4 +912,24 @@ describe('Catalogue Items Table', () => {
       expect(screen.getByDisplayValue('Details')).toBeInTheDocument();
     });
   });
+
+  it('renders table correctly (section 1 due to column virtualisation without spares)', async () => {
```

Comment:
```suggestion
  it('renders table correctly (section 1 due to column virtualisation) without spares', async () => {
```
To be consistent with test change above.